Extension/c_cpp_properties.schema.json

```diff
@@ -78,6 +78,9 @@
                             "msvc-x64",
                             "gcc-x64",
                             "clang-x64",
+                            "msvc-x86",
```

Comment:
We're going to need to update the cpptools API too.

Reply:
ok, also sort the elements like in the settings.html?

Reply:
it isn't necessary for the schema.